src/components/scene/canvas.js

```diff
@@ -29,6 +29,8 @@ module.exports.Component = register('canvas', {
       canvas.style.height = data.height + '%';
       canvas.style.width = data.width + '%';
       scene.appendChild(canvas);
+    } else {
+      canvas.dataset.aframeProvided = true;
```

Comment:
You should move it up to the `if` statement where the provided canvas is created.